model/licensing.py

```diff
@@ -206,6 +206,10 @@ def __repr__(self):
             self.licenses_reserved, self.patrons_in_hold_queue
         )
 
+    @property
+    def unlimited_access(self):
+        return self.licenses_owned is None
+
```

Comment:
This looks like you've added another situation where a book is considered available to be checked out. `Work.to_search_documents` needs to be changed so that an unlimited access book always counts as "available" and "licensed". Otherwise books of this sort won't show up when the end-user is filtering on books that are "available now".

Reply:
Added additional logic as requested